src/tests/benchmark_hdf5.c

```diff
@@ -71,8 +71,8 @@ int  debug;
  * timing(6) = Time to read connectivity table,          20,   21,    22
  * timing(7) = Time to read solution data (field data)   23,   24,    25
  * timing(8) = Time to read array data                   26,   27,    28
- * timing(9) = Time for cgp_open, CG_MODE_WRITE          29,   30,    31
- * timing(10) = Time for cgp_open, CG_MODE_READ          32,   33,    34
+ * timing(9) = Time for cg_open, CG_MODE_WRITE          29,   30,    31
```

Comment:
Looks like all these are just making the comments/output correct (`cg` vs `cgp`) and not changing from testing parallel to serial.  At first it looked like changing the test, but then I saw just correcting what was printed.